source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```
@@ -121,3 +121,51 @@
 		}
 	}
 }
+
+// A heading with an arrow and other content.
+.heading-with-arrow {
```

This should probably have some kind of prefix… maybe `wporg-pattern__`?

```suggestion
.wporg-pattern__heading-with-arrow {
```

---

I've been thinking about what prefix to use here... Thanks!